manifests/preflight.yaml

```diff
@@ -18,12 +62,38 @@ spec:
               when: ">= 1.15.0"
               message: Your cluster meets the recommended and required versions of Kubernetes.
     - customResourceDefinition:
-        customResourceDefinitionName: deployment
+        customResourceDefinitionName: ingressroutes.contour.heptio.com
+        outcomes:
+          - fail:
+              message: IngressRoutes CRD not found!
+          - pass:
+              message: IngressRoutes CRD found!
+    - containerRuntime:
         outcomes:
+          - pass:
+              when: "== docker"
+              message: Docker container runtime was found.
           - fail:
-              message: Kind deployment not found!
+              message: Did not find Docker container runtime.
+    - storageClass:
+        checkName: Required storage classes
+        storageClassName: "default"
+        outcomes:
+          - fail:
+              message: Could not find a storage class called default.
           - pass:
-              message: Deployment kind found!
+              message: All good on storage classes
+    # Does not even show up in preflights page.
```

Comment:
This one needs to have the secret included in a collector.  https://troubleshoot.sh/reference/analyzers/secrets/ . 

```
Collectors do not automatically include secrets because these often contain sensitive information. 
```

Reply:
Also, this still should be showing up with an error, I'll look into this